<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Service Controller</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pPushButtonStart_">
     <property name="text">
      <string>Sta&amp;rt:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pPushButtonInstall_">
     <property name="text">
      <string>&amp;Install</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pPushButtonUninstall_">
     <property name="text">
      <string>&amp;Uninstall</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pPushButtonPause_">
     <property name="text">
      <string>&amp;Pause</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="pPushButtonHttpGet_">
     <property name="text">
      <string>HTTP &amp;GET</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pPushButtonStatus_">
     <property name="text">
      <string>&amp;Status</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="pLineEditArgs_">
     <property name="toolTip">
      <string>Arguments</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="pSpinBoxCommandId_"/>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="pLineEditPassword_"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="pLineEditAccount_"/>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="pPushButtonSendCommand_">
     <property name="text">
      <string>S&amp;end Command:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Account:</string>
     </property>
     <property name="buddy">
      <cstring>pLineEditAccount_</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QPushButton" name="pPushButtonReloadConfiguration_">
     <property name="text">
      <string>Re&amp;load Configuration</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="pPushButtonStop_">
     <property name="text">
      <string>St&amp;op</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Pass&amp;word:</string>
     </property>
     <property name="buddy">
      <cstring>pLineEditPassword_</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="pPlainTextEditOutput_">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>285</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pPushButtonClear_">
        <property name="text">
         <string>&amp;Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QPushButton" name="pPushButtonResume_">
     <property name="text">
      <string>&amp;Resume</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pPushButtonStatus_</tabstop>
  <tabstop>pPushButtonInstall_</tabstop>
  <tabstop>pLineEditAccount_</tabstop>
  <tabstop>pLineEditPassword_</tabstop>
  <tabstop>pPushButtonUninstall_</tabstop>
  <tabstop>pPushButtonStart_</tabstop>
  <tabstop>pLineEditArgs_</tabstop>
  <tabstop>pPushButtonStop_</tabstop>
  <tabstop>pPushButtonPause_</tabstop>
  <tabstop>pPushButtonResume_</tabstop>
  <tabstop>pPushButtonSendCommand_</tabstop>
  <tabstop>pSpinBoxCommandId_</tabstop>
  <tabstop>pPushButtonReloadConfiguration_</tabstop>
  <tabstop>pPushButtonHttpGet_</tabstop>
  <tabstop>pPlainTextEditOutput_</tabstop>
  <tabstop>pPushButtonClear_</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
